freud/environment.pyx

```diff
@@ -137,6 +137,11 @@ cdef class BondOrder(Compute):
     def __dealloc__(self):
         del self.thisptr
 
+    @property
+    def default_query_args(self):
+        return dict(mode="nearest", r_max=self.r_max,
+                    num_neighbors=self.num_neighbors)
```

Comment:
Does it make sense for `self.r_max` to exist anymore? I think that once we're closer to finalizing the API it may make sense to make it possible for users to set the default query arguments (probably will require renaming to just `self.query_args` or so), but in general I don't think query argument-like things should be class members anymore unless they're strictly bounds on a histogram or something similar (like `r_max` for RDFs).

Reply:
Punting this for a later rewrite to support QueryArgs.